deeplake/core/tensor.py

```diff
@@ -1227,7 +1227,7 @@ def timestamps(self) -> np.ndarray:
             sub_index = index.values[1].value if len(index.values) > 1 else None
         global_sample_index = next(index.values[0].indices(self.num_samples))
         if self.is_link:
-            sample = self.chunk_engine.get_video_url(global_sample_index)  # type: ignore
+            sample = self.chunk_engine.get_video_url(global_sample_index)[0]  # type: ignore
```

Comment:
why these changes? looks like `get_video_url` returns a string

Reply:
have changed the method to return a tuple of url and path, path is needed for some error messaging as url could be a presigned one that the user won't recognize